dbt_project.yml

```diff
@@ -25,20 +25,60 @@ clean-targets:
+    # Please only add those that you change the values of to your root dbt_project.yml file, do not copy all values as this can lead to unexpected issues
+
+    # WAREHOUSE & TRACKERS
+    # See https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/ecommerce/ for more information
+    # snowplow__atomic_schema: "atomic_data_sample" # Only set if not using 'atomic' schema for Snowplow events data
+    # snowplow__database: Only set if not using target.database for Snowplow events data -- WILL BE IGNORED FOR DATABRICKS
+    snowplow__dev_target_name: 'dev'
     snowplow__events: "{{ source('atomic', 'events') }}"
-    snowplow__ecommerce_event_names: ['snowplow_ecommerce_action']
-    snowplow__number_checkout_steps: 4
-    snowplow__number_category_levels: 4
-    snowplow__categories_separator: '/'
-    snowplow__use_product_quantity: false
+    # snowplow__events_table: "events" # Only set if not using 'events' table for Snowplow events data
 
+    # OPERATION & LOGIC
+    snowplow__allow_refresh: false
+    snowplow__backfill_limit_days: 30
+    snowplow__categories_separator: '/'
+    snowplow__days_late_allowed: 3
     snowplow__enable_mobile_events: false
+    snowplow__lookback_window_hours: 6
+    snowplow__max_session_days: 3
+    snowplow__number_category_levels: 4
+    snowplow__number_checkout_steps: 4
+    snowplow__session_lookback_days: 730
+    snowplow__session_identifiers: []
+    # snowplow__session_sql: 'sc.session_id'  # takes priority over session_identifiers
+    snowplow__session_timestamp: collector_tstamp
+    snowplow__start_date: '2020-01-01'
+    snowplow__upsert_lookback_days: 30
+    snowplow__use_product_quantity: false
+    snowplow__user_identifiers: []
+    # snowplow__user_sql: 'sc.user_id' # takes priority over user identifiers
 
 
-    # Redshift/Postgres contexts
+    # CONTEXTS, FILTERS & LOGS
+    snowplow__app_id: []
+    snowplow__disable_ecommerce_cars: false
+    snowplow__disable_ecommerce_checkouts: false
+    snowplow__disable_ecommerce_products: false
+    snowplow__disable_ecommerce_transactions: false
+    snowplow__ecommerce_event_names: ['snowplow_ecommerce_action']
+    snowplow__carts_passthroughs: []
+    snowplow__checkouts_passthroughs: []
+    snowplow__products_passthroughs: []
+    snowplow__session_passthroughs: []
+    snowplow__transaction_passthroughs: []
+
+    # WAREHOUSE SPECIFIC
+      # BigQuery
+    snowplow__derived_tstamp_partitioned: true
+      # Snowflake
+    snowplow__query_tag: "snowplow_dbt"
+      # Databricks
+    # Depending on the use case it should either be the catalog (for Unity Catalog users from databricks connector 1.1.1 onwards) or the same value as your snowplow__atomic_schema (unless changed it should be 'atomic')
+    # snowplow__databricks_catalog: 'hive_metastore'
+      # Redshift/Postgres
```

Comment:
Flagging here but the position is not relevant. I tried testing for Redshift and I keep running into different errors. First I tried with dbt 1.6.1 but dbt seed failed due to `"'h' format requires -32768 <= number <= 32767"`, switched to 1.3.0 and at least then it ran fine. But then no matter which version I tried using, when I run the next step in the integration test I keep getting an error. Either: `get_column_schema_from_query is undefined `(not recognising dbt utils macro) using 1.3.0 or ```
atabase Error
  ('communication error', OSError('Int or String expected')) 
```using  1.5.4.

Reply:
I'll test with Redshift and try to solve this issue, thanks for flagging!

Reply:
So it looks like this issue is present in the `main` branch as well, and it might be related to seeding a certain amount of columns/data points? I think it comes from [here](https://github.com/dbt-labs/dbt-redshift/issues/347) but not really sure what we can do about it. The `get_column_schema_from_query` macro only exists in dbt v1.5+, but that's also when the data loading issue is introduced (I can't seed in any of the 1.5.X or 1.6.X versions).

Reply:
This is a seed related error, you need to add this macro to the int tests project when you run it.

```
{% macro get_batch_size() %}
  {{ return(300) }}
{% endmacro %}
```

Agnes is normally good at finding Slack threads! https://snowplow.slack.com/archives/D04M7K8NTRB/p1693385774509979

Reply:
Everything seems to pass on redshift